### PR TITLE
add accessibility help for debug console

### LINKED ISSUE #221603: add accessibility help dialog for debug console
Some things it should include:

- [x] An explanation to use arrow keys from input box to reach debug output
- [x] The setting `debug.expandLazyVariables`
- [x] Kbs for stepping over,  into, out of, and continue

The following should be added to the editor's accessibility help dialog:
- [x] Kb to set a breakpoint in the editor
- [x] The command which allows editor selection to be evaluated in the debug console, `editor.debug.action.selectionToRepl`, no keybinding assigned yet, maybe needs one

cc @jooyoungseo , @rperez030

### PR DESCRIPTION
**Pull request by meganrogge**
fixes https://github.com/microsoft/vscode/issues/221603

Also:
- ensures focus is returned to the input instead of the repl tree depending on what was initially focused
- adds important debugging info to the editor's accessibility help dialog

cc @roblourens

### CODE REVIEW

**Review by bhavyaus — APPROVED** [review]

File: src/vs/editor/common/standaloneStrings.ts [hunk 1]
```diff
@@ -26,6 +26,8 @@ export namespace AccessibilityHelpNLS {
 	export const listAlerts = nls.localize("listAnnouncementsCommand", "Run the command: List Signal Announcements for an overview of announcements and their current status.");
 	export const quickChat = nls.localize("quickChatCommand", "Toggle quick chat<keybinding:workbench.action.quickchat.toggle> to open or close a chat session.",);
 	export const startInlineChat = nls.localize("startInlineChatCommand", "Start inline chat<keybinding:inlineChat.start> to create an in editor chat session.");
+	export const setBreakpoint = nls.localize('debugConsole.setBreakpoint', "The Debug: Inline Breakpoint command<keybinding:editor.debug.action.toggleInlineBreakpoint> will set or unset a breakpoint at the current cursor position.");
```
Comment (roblourens) [inline src/vs/editor/common/standaloneStrings.ts:29]: You probably don't want the command ID or the special syntax to go through localization

Comment (meganrogge) [reply src/vs/editor/common/standaloneStrings.ts:29]: Hm, so would I pass those in as variables to the localization string?

Comment (roblourens) [reply src/vs/editor/common/standaloneStrings.ts:29]: You can use the {0} template syntax and pass in an unlocalized string

Comment (meganrogge) [reply src/vs/editor/common/standaloneStrings.ts:29]: #221843

**Pull request merged**